hazelcast/src/main/java/com/hazelcast/core/IdGenerator.java

```diff
@@ -42,31 +42,29 @@
  * <p>
  * Care should be taken if you are using the IdGenerator to provide keys for {@link IMap} or
  * {@link com.hazelcast.cache.ICache}, as after split brain it could be possible for two unique values to share
- * the same key.  When a {@link com.hazelcast.map.merge.MapMergePolicy} is applied, one of these unique values will
- * have to be discarded.  Unless configured the default merge policy is
+ * the same key. When a {@link com.hazelcast.map.merge.MapMergePolicy} is applied, one of these unique values will
+ * have to be discarded. Unless configured, the default merge policy is
  * {@link com.hazelcast.map.merge.PutIfAbsentMapMergePolicy}.
  * <p>
  * It is NOT RECOMMENDED to use {@link IdGenerator} to provide keys that are also used as unique identifiers in
- * underlying persistent storage, for example a {@link IMap} that writes to a Relational Database using {@link MapStore}.
+ * underlying persistent storage, for example in an {@link IMap} that writes to a relational database using {@link
+ * MapStore}.
```

Comment:
Should this interface be also `@deprecated` (since the relevant method to obtain a reference from `HazelcastInstance` is also `@deprecated`)? Or at least add a `@see` reference to the `ReliableIdGenerator` as a better alternative?